backend/api/Dockerfile

```diff
@@ -0,0 +1,28 @@
+FROM python:3.6.10-alpine
+
+WORKDIR /usr/src/stump_backend/api
+
+ENV PYTHONDONTWRITEBYTECODE 1
+ENV PYTHONUNBUFFERED 1
+
+# psycopg2 dependencies
+RUN apk update \
+  && apk add postgresql-dev gcc python3-dev musl-dev
+
+# install python dependencies
+COPY ./Pipfile .
+RUN pip install --upgrade pip \
+  && pip install pipenv \
+  && pipenv lock -r > ./requirements.txt \
```

Comment:
Why not just `pipenv sync` and use the existing Pipfile.lock, rather than generating a requirements.txt file?

Reply:
I've always used pip and venv separately rather than pipenv, and was just working with what I knew. I agree that ```pipenv sync``` is more concise. I'll make a commit to change it.

Reply:
@galbwe Yea, the workflow changed with `pipenv`. I am in the process of upgrading all my projects from Python2 to Python3 (pip and venv) and it is new to me. I am curious how well it works with docker builds.

Reply:
@coloradocarlos for ```Dockerfile.prod``` I may still need to generate a ```requirements.txt``` file in order to build wheels in the multistage build: https://github.com/pypa/pipenv/issues/1593